tests/package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
@lipmaneth `package.json` normally resides in the main project directory, not under `tests/`.